pkg/bootstrap/docker/down.go

```
@@ -64,6 +65,11 @@ func (c *ClientStopConfig) Stop(out io.Writer) error {
 		return err
 	}
 	helper := dockerhelper.NewHelper(client, nil)
+	glog.V(4).Infof("Killing previous socat tunnel")
+	err = openshift.KillExistingSocat()
+	if err != nil {
+		glog.V(1).Infof("error: cannot kill socat: %v", err)
```

should we abort at this point instead of continuing/eating the error?